pkgdb2client/cli.py

```diff
@@ -797,6 +810,30 @@ def do_monitoring(args):
             'messages', 'Invalid output returned, please contact an admin')
 
 
+def do_koschei(args):
+    ''' Retrieve or set the koschei monitoring status of a package from
+    pkgdb.
+
+    '''
+    LOG.info("package  : {0}".format(args.package))
+    LOG.info("koschei  : {0}".format(args.koschei))
```

Comment:
I used info everywhere, but it's not show by default, only on --verbose or --debug :)